guides/manual-installation.md

```diff
@@ -340,364 +304,323 @@ journalctl -fu trento-web
 
-```
+1. Test Trento web health status with curl:
+    ```bash
+    curl http://localhost:4000/api/readyz;
+    ```
+    ```bash
+    curl http://localhost:4000/api/healthz;
+    ```
 
-To check that everything is running as expected, you can run the following command:
+2.  Test Trento wanda health status with curl:
+    ```bash
+    curl http://localhost:4001/api/readyz
+    ```
+    ```bash
+    curl http://localhost:4001/api/healthz
+    ```
+    
 
-```bash
-docker ps
+Expected output if Trento web/wanda is ready and the database connection is setup correctly:
 ```
-
-Expected output:
-
-```bash
-CONTAINER ID   IMAGE                                         COMMAND                  CREATED          STATUS          PORTS                                       NAMES
-8b44333aec39   registry.suse.com/trento/trento-web:2.2.0    "/bin/sh -c '/app/bi…"   6 seconds ago    Up 5 seconds    0.0.0.0:4000->4000/tcp, :::4000->4000/tcp   trento-web
-e859c07888ca   registry.suse.com/trento/trento-wanda:1.2.0   "/bin/sh -c '/app/bi…"   18 seconds ago   Up 16 seconds   0.0.0.0:4001->4000/tcp, :::4001->4000/tcp   wanda
+{"ready":true}{"database":"pass"}
 ```
 
-Both containers should be running and listening on the specified ports.
-
-### Setup nginx as reverse proxy
-
-#### Option 1: Creating a Self-Signed Certificate
-
-> Note: This is a basic guide for creating a self-signed certificate for use with Trento. You may use your own certificate. For detailed instructions, consult the OpenSSL documentation. Remember to replace example.com with your domain.
-
-**Step 1**: Generate a private key:
-
-```bash
-openssl genrsa -des3 -out trento.key 2048
-```
-
-**Step 2**: Generate a CSR (Certificate Signing Request):
-
-```bash
-openssl req -new -key trento.key -out trento.csr
-```
-
-Follow the prompts to enter your information.
-
-**Step 3**: Remove passphrase from the key
-
-```bash
-cp trento.key trento.key.org
-openssl rsa -in trento.key.org -out trento.key
-```
-
-**Step 4**: Create config file for SAN:
-
-> Note: See [X509 V3 certificate extension configuration format documentation](https://www.openssl.org/docs/man1.0.2/man5/x509v3_config.html) for more details.
-
-```bash
-echo 'subjectKeyIdentifier = hash
-authorityKeyIdentifier = keyid:always,issuer:always
-basicConstraints = CA:TRUE
-keyUsage = digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment, keyAgreement, keyCertSign
-subjectAltName = DNS:example.com, DNS:*.example.com
-issuerAltName = issuer:copy' > v3.ext
-```
+## Prepare SSL certificate for NGINX
 
-**Step 5**: Generate and sign the certificate:
+Create or provide a certificate for [NGINX](https://nginx.org/en/) to enable SSL for Trento.
+This is a basic guide for creating a self-signed certificate for use with Trento. You may use your own certificate. For detailed instructions, consult the [OpenSSL documentation]((https://www.openssl.org/docs/man1.0.2/man5/x509v3_config.html)).
```

Comment:
Are these double parenthesis correctly rendered?

Reply:
Yeah funny, they are, but I will remove the duplicate :+1: